keyboards/levinson/keymaps/jyh/keymap.c

```diff
@@ -0,0 +1,232 @@
+#include "levinson.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```